app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Notifications.java

```diff
@@ -373,7 +373,7 @@ private void ArmTimer(Context ctx) {
         if (wakeIntent != null)
             alarm.cancel(wakeIntent);
         wakeIntent = PendingIntent.getService(this, 0, new Intent(this, this.getClass()), 0);
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
Hi All,
As we are not so certain of how to fix the Downtime issue, and the change
to Notifications.java didn't help in any way, I have backed out this change
in PebbleSync branch.
Should be fine to merge, though I might see if I can get the watch face
install in as well.  Might have some time today to finish that.
Cheers

On Thu, Jun 16, 2016 at 1:59 AM, tzachi-dar <notifications@github.com>
wrote:

> In
> app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Notifications.java
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/256#discussion_r67191138>
> :
>
> > @@ -373,7 +373,7 @@ private void ArmTimer(Context ctx) {
> >          if (wakeIntent != null)
> >              alarm.cancel(wakeIntent);
> >          wakeIntent = PendingIntent.getService(this, 0, new Intent(this, this.getClass()), 0);
> > -        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
> > +        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.LOLLIPOP) {
>
> Are you sure alarm.setAlarmClock should not be used in lollipop?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/256/files/a063e56a27b0e35ee4f36ddb0ed7af3f50a9d3a7#r67191138>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIQs80SPnDVLmoNpZLXSexTYp8XVLXH_ks5qMCFwgaJpZM4HHlcy>
> .
>



-- 
John Stevens
"You are how you live, not what you have."